packages/quantic/cypress/e2e/default-2/search-box/search-box.cypress.ts

```diff
@@ -0,0 +1,750 @@
+          beforeEach(() => {
+            setRecentQueriesInLocalStorage([]);
+            visitSearchBox({...defaultOptions, textarea});
+            mockQuerySuggestions(exampleQuerySuggestions);
+          });
+
+          it('should display the suggestions in the right order and should not display recent query suggestions', () => {
+            scope('when loading standalone search box', () => {
+              Expect.displayInputSearchBox(true, textarea);
+              Expect.displaySearchButton(true);
+            });
+
+            scope('when focusing on the search box input', () => {
+              Actions.focusSearchBox(textarea);
+              cy.wait(InterceptAliases.QuerySuggestions);
+
+              Expect.displaySuggestionList(true);
+              Expect.displayClearRecentQueriesButton(false);
+              Expect.numberOfQuerySuggestions(exampleQuerySuggestions.length);
+              Expect.querySuggestionsEquals(exampleQuerySuggestions);
+            });
+
+            scope('when selecting a suggestions', () => {
+              const clickedSuggestionIndex = 0;
+              Actions.clickQuerySuggestion(clickedSuggestionIndex);
+
+              Expect.searchWithQuery(
+                exampleQuerySuggestions[clickedSuggestionIndex],
+                {
+                  LSkey: recentQueriesLSKey,
+                  queries: [exampleQuerySuggestions[clickedSuggestionIndex]],
+                }
+              );
+              Expect.logClickSuggestion(
+                exampleQuerySuggestions[clickedSuggestionIndex]
+              );
+            });
+          });
+        });
+
+        describe('when recent queries are found at initial state', () => {
+          const exampleRecentQueries = ['foo', 'bar'];
+
+          beforeEach(() => {
+            setRecentQueriesInLocalStorage(exampleRecentQueries);
+            visitSearchBox({...defaultOptions, textarea});
+            mockQuerySuggestions(exampleQuerySuggestions);
+          });
+
+          it('should display the suggestions and the recent query suggestions in the right order', () => {
+            const expectedQuerySuggestions = [
+              ...exampleRecentQueries,
+              ...exampleQuerySuggestions,
+            ];
+            scope('when loading standalone search box', () => {
+              Expect.displayInputSearchBox(true, textarea);
+              Expect.displaySearchButton(true);
+            });
+
+            scope('when focusing on the search box input', () => {
+              Actions.focusSearchBox(textarea);
+              cy.wait(InterceptAliases.QuerySuggestions);
+
+              Expect.displaySuggestionList(true);
+              Expect.displayClearRecentQueriesButton(true);
+              Expect.numberOfQuerySuggestions(expectedQuerySuggestions.length);
+              Expect.querySuggestionsEquals(expectedQuerySuggestions);
+            });
+
+            scope('when selecting a recent query', () => {
```

Comment:
why this scope block here if we only test that the recent queries/suggestions are displayed in the right order?

Reply:
to make sure that if we click the first option in that oder we will click a suggestion and long the correct analytics.